app/admin/textrepl/template.hbs

```diff
@@ -0,0 +1,35 @@
+<div class="panel panel-primary">
+<h3>{{t 'admin.textrepl.existing_repl'}}</h3>
+<p>{{t 'admin.textrepl.repl_desc'}}</p>
+<table class="table">
+  <tr class="table-header">
+    <th style="min-width: 150px">{{t 'labels.from'}}</th>
```

Comment:
Yes, please no inline styles. Add a class and then style the class in an existing or new SCSS file please. :bow: